application/classes/Ushahidi/Core.php

```diff
@@ -467,7 +467,9 @@ public static function init()
 		$di->set('repository.post.point', $di->lazyNew('Ushahidi_Repository_Post_Point'));
 		$di->set('repository.post.relation', $di->lazyNew('Ushahidi_Repository_Post_Relation'));
 		$di->set('repository.post.text', $di->lazyNew('Ushahidi_Repository_Post_Text'));
+		$di->set('repository.post.description', $di->lazyNew('Ushahidi_Repository_Post_Description'));
```

Comment:
I thought your plan was to still save title and description into the posts table? or did that not work as well as expected?